packages/react-composites/review/react-composites.api.md

```diff
@@ -539,8 +569,15 @@ export interface CallWithChatCompositeStrings {
 
 // @beta
 export interface CallWithChatControlOptions extends Pick<CallControlOptions, 'cameraButton' | 'microphoneButton' | 'screenShareButton' | 'displayType'> {
```

Comment:
I'm confused. So we now have *both* `extends Pick<CallControlOptions, 'cameraButton'...` and an explicit `cameraButton?: boolean` field? Worse, the two can diverge without build failure.

What I was expecting here:
* Drop the `Pick<...>` entirely.
* Add internal build-time check that `CallWithChatControlOptions` indeed extends `Pick <..>` etc.

Reply:
oh nope, that was not removed by accident! but I really like that internally we still use Pick, I will update with that

Reply:
See https://github.com/Azure/communication-ui-library/pull/1560/files#diff-db700faf40319449d13fa41bfdf20a7dc853a55c7f6a8e6f8dd3bb09d85a51d5 for an example.